index.html

```diff
@@ -200,6 +200,7 @@
       <li>– <a href="#Chainable-defaultTo">defaultTo</a></li>
       <li>– <a href="#Chainable-unsigned">unsigned</a></li>
       <li>– <a href="#Chainable-notNullable">notNullable</a></li>
+      <li>– <a href="#Chainable-notNull">notNull</a></li>
```

Comment:
I didn't see any documentation block for this. Looks like incomplete doc addition.